src/Cache/CacheClient.php

```diff
@@ -1899,7 +1899,84 @@ public function sortedSetFetchByRank(string $cacheName, string $sortedSetName, ?
         return $this->sortedSetFetchByRankAsync($cacheName, $sortedSetName, $startRank, $endRank, $order)->wait();
     }
 
-    // placeholder: sortedSetFetchByScore
+    /**
+     * Fetch the elements in a sorted set by score.
+     *
+     * @param string $cacheName Name of the cache that contains the set.
+     * @param string $sortedSetName The sorted set to fetch.
+     * @param ?float $minScore The minimum score (inclusive) of the
+     * elements to fetch. Defaults to negative infinity.
+     * @param ?float $maxScore The maximum score (inclusive) of the
+     * elements to fetch. Defaults to positive infinity.
+     * @param ?int $order The order to fetch the elements in. Defaults to Ascending.
+     *  Will be treated as ascending if SORT_ASC or higher is supplied, and descending if SORT_DESC or lower is supplied.
```

Comment:
What does "or higher" and "or lower" mean here?

Reply:
order is an integer. SORT_ASC is 4 and SORC_DESC is 3. you can supply any integer value. We're saying that if you supply 3 or less it is descending and if you supply 4 or more it is ascending.